### PR TITLE
Add upgrade page

### PR DESCRIPTION
**Pull request by johnkuney**
Creating v2 page for /upgrade. Carries over same info as legacy page plus additional link to blog post as requested

### CODE REVIEW

File: v2/src/pages/upgrade.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+          The Bitcoin Cash network will undergo a protocol upgrade as per the
+          roadmap. Businesses and individuals who use the Bitcoin Cash network
+          should check to ensure that their software is compatible with the
+          upgrade.
+        </p>
+        <PrimaryButton
+          noMarginLeft={true}
+          buttonText={fbt(
+            "See Roadmap",
+            "'See Roadmap' button of the annoucement bar"
```
Comment (deadalnix) [inline v2/src/pages/upgrade.js:45]: It's not the announcement bar :)

File: v2/src/pages/upgrade.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  return (
+    <>
+      <SEO title={fbt("How to Upgrade", "Upgrade page SEO title")} />
+      <Container className={S.linkcontainer}>
+        <h2 className="centerh2">
+          <fbt desc="Upgrade page heading">How to Upgrade</fbt>
+        </h2>
+        <h3>{upgradeTimeStr}</h3>
+        <h3>
+          <fbt desc="Annoucement bar inner headline">
```
Comment (deadalnix) [inline v2/src/pages/upgrade.js:31]: dito

File: v2/src/pages/upgrade.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+const UpgradePage = () => {
+  const locale = useLocaleContext()
+  const upgradeTimeStr = new Date(
+    ACTIVATION_TIMESTAMP * 1000
+  ).toLocaleDateString(locale.bcp47, {
+    year: "numeric",
+    month: "short",
+    day: "numeric",
+  })
```
Comment (deadalnix) [inline v2/src/pages/upgrade.js:20]: Don't duplicate logic, factorize instead.

**Review by deadalnix — CHANGES_REQUESTED** [review]

File: v2/src/pages/upgrade.js (hunk 1 above)
Comment (johnkuney) [reply v2/src/pages/upgrade.js]: ah for sure, changed

File: v2/src/pages/upgrade.js (hunk 2 above)
Comment (johnkuney) [reply v2/src/pages/upgrade.js]: fixed

File: v2/src/pages/upgrade.js (hunk 3 above)
Comment (johnkuney) [reply v2/src/pages/upgrade.js]: yeah good call. Made a global helper for this and added to the announcement bar as well. That work?

**Pull request merged**